charters/UploadService/charter.md

```diff
@@ -0,0 +1,80 @@
+# Upload Service
+
+## Description
+The Human Cell Atlas (HCA) Data Coordination Platform (DCP) Upload Service provides a file staging and validation facility. Upload Areas are created and deleted using a [REST](https://en.wikipedia.org/wiki/Representational_state_transfer) Application Programming Interface (API), which is secured so only the DCP Ingestion Service may use it. It stages files into cloud storage and computes checksums for the files. The validation service can run user-defined validation jobs against files.
```

Comment:
This charter mentions "user-defined" validation jobs. I am not sure if that is submitters (scientists) or internal users. I assume it is possible for anyone to offer a validation (nice engineering design) but we would have project standard validation made by internal users. Could this be clarified in the language of the charter?